### PR TITLE
Feature/splash

### LINKED ISSUE #140: Remove white launch screen
There's a brief white "react native" splash screen that shows on launch. We should replace that with our own splash screen

### LINKED ISSUE #147: Lock screen rotation: portrait mode only
like the old app, lock the screen rotation to portrait only.

IMG_95BC00068CC7-1

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
Shows a splash screen until the initial data fetch completes. Also, locks the app to portrait mode. Planning to address the security screen in a separate PR.

Closes #140 
Closes #147

### CODE REVIEW

File: ios/Helium/Info.plist [hunk 1]
```diff
@@ -37,22 +37,18 @@
...
 	</dict>
-	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>Allow Helium to use your location to add your Helium Hotspots to the network</string>
-	<key>NSFaceIDUsageDescription</key>
-	<string>Allow Helium to authenticate with Face ID</string>
 	<key>NSBluetoothAlwaysUsageDescription</key>
 	<string>Allow Helium to connect to and manage your Hotspots</string>
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Allow Helium to connect to and manage your Hotspots</string>
-	<key>UIBackgroundModes</key>
```
Comment (tyler-whitman) [inline ios/Helium/Info.plist:48]: Did we want to remove this? I think it was for push notifications?

File: ios/Podfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 require_relative '../node_modules/react-native-unimodules/cocoapods.rb'
 
-platform :ios, '10.0'
+platform :ios, '11.0'
```
Comment (tyler-whitman) [inline ios/Podfile:5]: Intended to update this?

File: src/App.tsx [hunk 3]
```diff
@@ -18,25 +18,37 @@ import { useAsync } from 'react-async-hook'
...
 
 const App = () => {
   if (Platform.OS === 'android') {
     if (UIManager.setLayoutAnimationEnabledExperimental) {
       UIManager.setLayoutAnimationEnabledExperimental(true)
     }
-    LogBox.ignoreLogs(['Setting a timer'])
   }
 
+  LogBox.ignoreLogs([
```
Comment (tyler-whitman) [inline src/App.tsx:44]: Oh nice, I added this in the index file too but left it commented out. Ill remove mine and we can just merge yours in

File: src/App.tsx [hunk 4]
```diff
@@ -80,17 +93,43 @@ const App = () => {
     [dispatch],
   )
 
-  useEffect(() => {
-    if (!isRestored && !isBackedUp) return
+  const loadInitialData = useCallback(async () => {
+    dispatch(restoreUser())
+    dispatch(fetchInitialData())
```
Comment (tyler-whitman) [inline src/App.tsx:98]: I noticed this is calling to get the oracle price and current block height. Do we need that data before anything else in the app can load?

Its not too much, but in the old app we slowly got way too many things loading at the start and it was a really long launch process haha, so just want to be sure we dont just dump calls here.

**Review by tyler-whitman — APPROVED** [review]

File: ios/Podfile (hunk 2 above)
Comment (matthewcarlreetz) [reply ios/Podfile:5]: yeah, the expo splash library required it.

File: src/App.tsx (hunk 4 above)
Comment (matthewcarlreetz) [reply src/App.tsx:98]: yeah, I hear what you're saying. @allenan is addressing data loading in a separate PR. Going to leave it as is for now.

File: ios/Helium/Info.plist (hunk 1 above)
Comment (matthewcarlreetz) [reply ios/Helium/Info.plist:48]: No, whoops, I don't know why these things got removed. I just intended to remove the landscape modes.

**Pull request merged**